vcf-dumper/vcf-dumper-ws/src/main/java/uk/ac/ebi/eva/vcfdump/server/VcfDumperWSConfig.java

```diff
@@ -34,6 +34,8 @@ public ThreadPoolTaskExecutor mvcAsyncThreadPool() {
     @Override
     public void configureAsyncSupport(AsyncSupportConfigurer configurer) {
         configurer.setTaskExecutor(mvcAsyncThreadPool());
+        long milliseconds = 300000L;
+        configurer.setDefaultTimeout(milliseconds);
```

Comment:
Did the property `spring.mvc.async.request-timeout` not work?

Reply:
I couldn't make it work